pkg/tinyvcdiff/doc.txt

```diff
@@ -0,0 +1,15 @@
+/**
+ * @defgroup pkg_tinyvcdiff   Tiny VCDIFF
+ * @ingroup  pkg
+ * @brief    Decoder for interleaved VCDIFF deltas
+ *
+ * # Introduction
+ *
+ * Put an introduction of the package here
```

Comment:
This generated line needs to be updated ;)

Reply:
I've written some lines how to use this lib.